packages/server/events-processing-platform/subscriptions/organization/organization_event_handler.go

```diff
@@ -195,6 +198,21 @@ func (h *organizationEventHandler) webScrapeOrganization(ctx context.Context, te
 	return nil
 }
 
+func (h *organizationEventHandler) updateOrganizationNameIfEmpty(ctx context.Context, tenant, url string, organization *entity.OrganizationEntity, span opentracing.Span) {
+	if organization.Name == "" && strings.Contains(url, ".") {
```

Comment:
although I think the code will work as intended, this part would fail if for any reason we had a bug or failure and got `www.subdomain.domain.com` or `www.domain` for a value rather than `domain.com`, wouldn't perhaps a more thorough check help avoid this? (e.g.: checking it has two parts only, or `.com` at the end, or that there are no subdomain or `www` ?)